var/spack/repos/builtin/packages/audacious/package.py

```diff
@@ -27,6 +26,14 @@ class Audacious(AutotoolsPackage):
     depends_on('glib')
     depends_on('qt')
 
+    def patch(self):
+        search_path_args = ' '.join(self.autoreconf_search_path_args)
+        filter_file(
+            '-I m4',
+            '-I m4 {0}'.format(search_path_args),
+            os.path.join(self.stage.source_path, 'autogen.sh')
```

Comment:
```suggestion
            'autogen.sh'
```

Reply:
I used the absolute path to be independent of the working directory. Was that giving issues or not working correctly?

Reply:
But patch's working directory is the source path.
https://spack.readthedocs.io/en/latest/packaging_guide.html#working-dir
I think we don't need to specify absolute path.

Reply:
Fair enough. I think changing the working directory of the `patch` method is unlikely to happen in the future. I'll do as requested and remove the import. Thanks.

Reply:
Done in 3d2144a